kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/Suppliers.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.good.patterns.food2door;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Suppliers {
+    List<Supplier> suppliers = new ArrayList<>();
+    public void addSupplier(Supplier supplier){
+        suppliers.add(supplier);
+    }
+    public List supplierProducts(Supplier supplier){
```

Comment:
consider changing name of this method to getProductsFor, then in code you will write suppliers.getProductsFor(givenSupplier)
Also, addSupplier can be simplified to just add, and then you write suppliers.add(concreteSuplier)